### ISSUE TITLE
Where is `.csv` file

### ISSUE THREAD

**Issue by Chrisleowoo** [WeixiongLin/PMC-CLIP#16]
Thanks for presenting this amazing work!

On reproduction, I encountered with some problem: python -m training.main --dataset-type "csv" --csv-separator "," --save-frequency 5 --report-to tensorboard --train-data="path/to/train.csv" --val-data="path/to/valid.csv" --csv-img-key image --csv-caption-key caption --warmup 500 --batch-size=8 --lr=1e-4 --wd=0.1 --epochs=100 --workers=8 --model RN50_fusion4 --hugging-face --mlm --crop-scale 0.5

in the command, where is the csv filepath you used in pretraining ? I can't find then in whole directory here...

Appreciate your response!

**Comment by Asunatan** [comment]
+1

**Comment by WeixiongLin** [comment]
Thanks for your interest. You can download the dataset as I described in README.
> Download from [Huggingface](https://huggingface.co/datasets/axiong/pmc-oa), [Baidu Cloud](https://pan.baidu.com/s/1mD51oOYbIOqDJSeiPNaCCg)(key: 3iqf).
Or follow the [Pipeline of PMC-OA Development](https://github.com/WeixiongLin/Build-PMC-OA) if you want to start from scratch.

Feel free to reopen the issuee if the problem is still there.

**Comment by PixelChen24** [comment]
> 3iqf

In the files list of both huggingface and baidu cloud, there is no such a file ending with csv
